refresher/pygmt_tutorial.ipynb

```diff
@@ -0,0 +1,677 @@
+    "call a plotting method, you will probably want to use\n",
+    "these three parameters:\n",
+    "\n",
+    "- The ``frame`` parameter.\n",
+    "  - This parameter controls the appearance of the map frame\n",
+    "    boundary.\n",
+    "  - For simple, automatic frame settings you can use ``frame=True``.\n",
+    "  - There's a lot the frame parameter can do. We'll learn more\n",
+    "    later, or you can check out the [frame tutorial](https://www.pygmt.org/v0.4.1/tutorials/frames.html).\n",
+    "- The ``region`` parameter.\n",
+    "  - This parameter controls the boundaries of the plot, usually\n",
+    "    by accepting an array in the form `[xmin, xmax, ymin, ymax]`.\n",
+    "  - There are a few special options for the ``region`` parameter,\n",
+    "    which you can learn about in the [region tutorial](https://www.pygmt.org/v0.4.1/tutorials/regions.html).\n",
+    "- The ``projection`` parameter.\n",
+    "  - This parameter controls how your data are mapped onto a 2D\n",
+    "    plot. The control over projection is what sets libraries\n",
+    "    like PyGMT and Cartopy apart from libraries focused on\n",
+    "    visualizing non-geographic datasets. There are many projections\n",
+    "    options listed in the [projections table](https://www.pygmt.org/v0.4.1/projections/index.html#projection-table)\n",
+    "    with examples shown in the [projections gallery](https://www.pygmt.org/v0.4.1/projections/index.html).\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a8c33655-5ce9-4776-b60f-b5135f882f89",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Make a quick plot of the data\n",
+    "# Create an instance of the pygmt.Figure class\n",
+    "fig = pygmt.Figure()\n",
+    "# Plot the data on the figure using 1 point, blue circles (`style=\"c1p\", color=\"slateblue\"`)\n",
+    "# Set a simple, automatic frame (`frame=True`)\n",
+    "# Use a Mercator projection on a 15 cm wide plot (`projection=\"M15c\"`)\n",
+    "fig.plot(\n",
+    "    data=data,\n",
+    "    frame=True,\n",
+    "    region=region,\n",
+    "    projection=\"M15c\",\n",
+    "    style=\"c1p\",\n",
+    "    color=\"slateblue\",\n",
+    ")\n",
+    "# Display the figure\n",
+    "fig.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e1453bb8-c656-4cb1-b98a-2e0dd9843607",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Plot tabular data with symbols colored by data values\n",
+    "We'll use the same [pygmt.Figure.plot](https://www.pygmt.org/v0.4.1/api/generated/pygmt.Figure.plot.html)\n",
+    "method with the symbol color depending on the z-value, by using\n",
+    "[pygmt.makecpt](https://www.pygmt.org/v0.4.1/api/generated/pygmt.makecpt.html) to\n",
+    "create a colormap using one of the\n",
+    "[Scientific Colour Maps](https://www.fabiocrameri.ch/colourmaps/)\n",
+    "available through GMT and setting ``cmap=True`` in ``pygmt.Figure.plot``."
```

Comment:
Recommend to link to the list of built-in CPTs at https://docs.generic-mapping-tools.org/6.2/cookbook/cpts.html#id3.

```suggestion
    "available through GMT and setting ``cmap=True`` in ``pygmt.Figure.plot``.\n",
    "The full list of built-in colourmaps in GMT, see\n",
    "https://docs.generic-mapping-tools.org/6.2/cookbook/cpts.html#id3."
```